server/secret.go

```diff
@@ -1,32 +1,46 @@
 package server
 
 import (
+	"net/http"
+
 	"github.com/drone/drone/model"
 	"github.com/drone/drone/router/middleware/session"
 	"github.com/drone/drone/store"
 
 	"github.com/gin-gonic/gin"
 )
 
+func GetSecrets(c *gin.Context) {
+	repo := session.Repo(c)
+	secrets, err := store.GetSecretList(c, repo)
+
```

Comment:
let's convert secrets to the below type before we write to the response. This way we can prevent the plaintext password from being returned in the response.

```
type secretResp struct {
	ID int64 `json:"id"`
	Name string `json:"name"`
	Images []string `json:"image,omitempty"`
	Events []string `json:"event,omitempty"`
}
```